### DISCUSSION TITLE
컨벤션 모음을 위키에 작성했습니다.

### DISCUSSION THREAD

**Discussion by gomiseki** [FE-CheckList-Gallery/FE-CheckList-Gallery#3] [Announcements]
제가 이전 프로젝트에서 썼던 컨벤션에 다른 조들 컨벤션을 참조해서 얹어봤는데요,
action 종류라던가 분류하는데 너무 고민하지 않고 자율적으로 판단해서 사용하면 될 거 같습니다.

질문 혹은 추가되어야 할 컨벤션이 있거나 수정이 필요하다고 생각되면 바로 말씀해주세요.
특히 깃허브의 fork 방식은 Organization에서는 fork가 낫지 않을까 고민해서 나온 내용입니다.
엘리스에서는 제대로 나오지 않는 내용이기에 익숙한 방식이 더 좋지 않을까 생각하시면 바로 말씀해주세요.

**Comment by ycs1m1yk** [comment]
컨벤션을 위키에 써놓는거 좋네요. 작성해주셔서 감사합니다!

위키 읽고서 몇가지 의견 남겨봅니다.

### 컨벤션 관련 레퍼런스 추가

주현님이 컨벤션의 큰 틀은 작성해주셨지만

모든 case에 대해 명세할 수는 없기 때문에

참고하기 좋은 레퍼런스가 있으면 좋을 것 같습니다.

ex) 컨벤션 관련 포스트, 유명한 깃허브 레포지토리

커밋 메세지 관련 저는 아래의 목록을 참고했습니다.
+ https://meetup.toast.com/posts/106
  좋은 git 커밋 메시지를 작성하기 위한 7가지 약속
+ https://github.com/microsoft/vscode/commits/main
  vscode 레포지토리의 커밋메세지 살펴보기
+ https://github.com/tensorflow/tensorflow/commits/master
  tensorflow 레포지토리의 커밋메세지 살펴보기

### 커밋메세지 header 형식

 1. [action] ~~ (현재)
  `[feat] eslint세팅 및 lint fix`

 2. Action: ~~
 `Feat: eslint세팅 및 lint fix`

이건 개인취향인데 저는 2번 형식을 선호해서 의견 제시해봅니다.

**Reply by SeolJaeHyeok** [reply]
좋은 것 같습니다! 
저도 2번 형식을 선호해서 지난 팀 프로젝트에서도 2번을 주로 썼던 기억이 있습니다:)

++
유명 오픈소스 커밋 메세지를 참고하는 것도 좋은 것 같아요!
